presto-product-tests/src/main/java/io/prestosql/tests/TestGroups.java

```diff
@@ -64,6 +64,7 @@
     public static final String HIVE_TABLE_STATISTICS = "hive_table_statistics";
     public static final String KAFKA = "kafka";
     public static final String AVRO = "avro";
+    public static final String HIVE_ACID = "hive_acid";
```

Comment:
I will merge these to hive group while removing the new product tests profile. All this is from the time when default product tests werent running on hive3.

Reply:
I checked the current state of product tests, I was wrong that tests have moved to hive3 image, I see that they are still running on hdp2.6-hive.
ACID tests cannot run without hdp3 image. If we dont differentiate these tests and put it under one of the existing test groups, the tests would fail when run with default hdp2.6-hive image.

Reply:
I have retained this part and travis tests wouldnt pass otherwise.